flash/text/seq2seq/question_answering/data.py

```diff
@@ -29,19 +94,143 @@ def __init__(
         max_target_length: int = 128,
         padding: Union[str, bool] = 'max_length'
     ):
+        self.backbone = backbone
+        self.max_target_length = max_target_length
+        self.max_source_length = max_source_length
+        self.padding = padding
+
         super().__init__(
             train_transform=train_transform,
             val_transform=val_transform,
             test_transform=test_transform,
             predict_transform=predict_transform,
-            backbone=backbone,
-            max_source_length=max_source_length,
-            max_target_length=max_target_length,
-            padding=padding,
+            data_sources={
+                DefaultDataSources.CSV: Seq2SeqCSVDataSource(
+                    self.backbone,
+                    max_source_length=max_source_length,
+                    max_target_length=max_target_length,
+                    padding=padding,
+                ),
+                DefaultDataSources.JSON: Seq2SeqJSONDataSource(
+                    self.backbone,
+                    max_source_length=max_source_length,
+                    max_target_length=max_target_length,
+                    padding=padding,
+                ),
+                "sentences": Seq2SeqSentencesDataSource(
+                    self.backbone,
+                    max_source_length=max_source_length,
+                    max_target_length=max_target_length,
+                    padding=padding,
+                ),
```

Comment:
Should we remove these? Do they work for QA? Maybe just remove the sentences one if the CSV and JSON would still work.

Reply:
I think we can remove the sentences data source.